include/deal.II/base/symmetric_tensor.h

```diff
@@ -2916,7 +2916,9 @@ operator * (const SymmetricTensor<rank,dim,Number> &t,
   // (as well as with switched arguments and double<->float).
   typedef typename ProductType<Number,OtherNumber>::type product_type;
   SymmetricTensor<rank,dim,product_type> tt(t);
-  tt *= product_type(factor);
+  product_type new_factor;
+  new_factor = factor;
```

Comment:
This is because `VectorizedArray<Number>` does have a copy-assignment operator from `Number` but no copy-construction method from `Number`. The latter is not allowed in the strict definition of a POD (plain-old-array) type.